library/java/org/chromium/net/impl/CronetBidirectionalStream.java

```diff
@@ -653,13 +653,13 @@ private void onErrorReceived(int errorCode, EnvoyFinalStreamIntel finalStreamInt
       mResponseInfo.setReceivedByteCount(finalStreamIntel.getReceivedByteCount());
     }
 
-    NetError netError = mapEnvoyMobileErrorToNetError(finalStreamIntel.getResponseFlags());
+    NetError netError = mapEnvoyMobileErrorToNetError(finalStreamIntel);
     int javaError = mapNetErrorToCronetApiErrorCode(netError);
 
     if (isQuicException(javaError)) {
       mException.set(new QuicExceptionImpl("Exception in BidirectionalStream: " + netError,
                                            javaError, netError.getErrorCode(),
-                                           /*nativeQuicError*/ 0));
+                                           /*nativeQuicError*/ 1));
```

Comment:
What does 1 mean in this context? Could we perhaps create a named constant and use that instead? (Here and below)

Reply:
Done